dandischema/models.py

```diff
@@ -903,14 +973,18 @@ def contributor_musthave_contact(cls, values):
         min_items=1,
     )
     dateCreated: Optional[datetime] = Field(
-        None, nskey="schema", title="Dandiset creation date and time", readOnly=True
+        None, nskey="schema", title="Dandiset creation date and time.", readOnly=True
     )
     dateModified: Optional[datetime] = Field(
-        None, nskey="schema", title="Last modification date and time", readOnly=True
+        None, nskey="schema", title="Last modification date and time.", readOnly=True
     )
 
     license: List[LicenseType] = Field(
-        min_items=1, description="Licenses associated with the item.", nskey="schema"
+        min_items=1,
+        description="Licenses associated with the item. DANDI only supports a "
+        "subset of Creative Commons Licenses (creativecommons.org) "
+        "relevant to datasets.",
```

Comment:
```suggestion
        "applicable to datasets.",
```

no way to borrow it from another class/item?